docs/detections/alerts-view-details.asciidoc

```diff
@@ -14,6 +14,12 @@ The alert details flyout contains these informational tabs:
 * *JSON*: The alert data in JSON format.
 * *Osquery Results*: Results from queries attached to rules display on the *Osquery Results* tab. <<add-osquery-response-action,Learn more.>>
 
+The **Share alert** link gives you a direct way to share alert details. Click the link to get a shareable URL that you can paste into browsers, cases, messages, and more. We don't recommend using the general browser URL to share the alert. It can cause inconsistent results if the Alerts page has filters or relative time ranges. 
+
+IMPORTANT: If you've enabled grouping on the Alerts page, the alert details flyout won't open until you expand the collapsed group. 
+
+NOTE: If you've set the {kibana-ref}/settings.html#server-publicBaseUrl[`server.publicBaseUrl`] configuration setting in the `kibana.yml` file, you can also find the shareable URL in the `kibana.alert.url` field. You can find the field by going the *Table* tab and searching for `kibana.alert.url`.
```

Comment:
A few nit edit suggestions for wordiness, repetition, etc. Also, "direct way" and "general browser URL" weren't super clear to me, so added some tweaks.
```suggestion
The **Share alert** link gives you a way to share alert details. Click the link to copy a URL that you can paste into browsers, cases, messages, and more. We recommend doing this instead of copying the URL from your browser's address bar, which can cause inconsistent results if the Alerts page has filters or relative time ranges. 

IMPORTANT: If you've enabled grouping on the Alerts page, the alert details flyout won't open until you expand the collapsed group. 

NOTE: If you've set the {kibana-ref}/settings.html#server-publicBaseUrl[`server.publicBaseUrl`] configuration setting in the `kibana.yml` file, the shareable URL is also in the `kibana.alert.url` field. You can find the field by searching for `kibana.alert.url` on the *Table* tab.
```